executors/cpp/datetime_fmt.cpp

```diff
@@ -180,9 +185,10 @@ const string TestDatetimeFmt(json_object *json_in) {
     return json_object_to_json_string(return_json);
   }
 
-  if (tz) {
-    df->setTimeZone(*tz);
-  }
+  // !!! IS OFFSET ALREADY CONSIDERED?
+  // if (tz) {
+  //   df->setTimeZone(*tz);
+  // }
```

Comment:
I think you need to set the `tz` somewhere, or else you won't be formatting in a time zone.